src/app/connections/list-toolbar/list-toolbar.component.html

```diff
@@ -11,11 +11,18 @@
             <button type="button" class="btn btn-default dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Name&nbsp;<span class="caret"></span>
               </button>
             <ul class="dropdown-menu">
-              <li><a href="#">Name</a></li>
-              <li><a href="#">Type</a></li>
+              <li><a (click)="setPropertyName('name')">Name</a></li>
+              <!--
+                TODO 'type' is not a valid property currently
+              <li><a (click)="setPropertyName('type')">Type</a></li>
+              -->
             </ul>
           </div>
-          <input type="text" class="form-control" placeholder="Filter By Name..." autocomplete="off" id="filterInput">
+          <input #filterInput 
```

Comment:
So for now we are not moving this to a `shared` component, I presume? We obviously have priorities but I would really like to have this available as a reusable component from a common area.